bug_report_templates/test-ransack-scope-and-column-same-name.rb

```diff
@@ -39,7 +39,7 @@
 # Display versions.
 message = "Running test case with Ruby #{RUBY_VERSION}, Active Record #{
   ::ActiveRecord::VERSION::STRING}, Arel #{Arel::VERSION} and #{
-  ::ActiveRecord::Base.connection.adapter_name}"
+  ::ActiveRecord::Base.adapter_class::ADAPTER_NAME}"
```

Comment:
I've opted to use this constant even where we could lease a connection (i.e., here and tests) for consistency.